ledger/accountdb.go

```diff
@@ -2567,12 +2576,7 @@ func (qs *accountsDbQueries) listCreatables(maxIdx basics.CreatableIndex, maxRes
 	return
 }
 
-type persistedValue struct {
```

Comment:
I just renamed and moved this but I don't mind undoing it if there's an objection.

Reply:
@algoidurovic I _think_ the rename touches on a central question:  What is box-specific about the cache?

I realize the question prompts a potentially large renaming exercise, but I'm wondering if the cache ought to be _kv_ themed rather than _box_ themed.

Reply:
yes, I think it should.  Anything below the ledger tries to talk about kv rather than about boxes. But I don't think it changes much code.

Reply:
@algoidurovic From my perspective, based on renames to _KV_, the thread is now resolved.  Since I didn't open it, I'm leaving it open for you to resolve.